sns/cm/storage.c

```diff
@@ -197,13 +197,17 @@ static int cob_stob_check(struct m0_cm_cp *cp)
 	struct m0_sns_cm_file_ctx *fctx;
 	struct m0_fid             *pver;
 	struct m0_cob             *cob;
+	int                        ret;
 
 	fctx = ag2snsag(cp->c_ag)->sag_fctx;
 	pver = &fctx->sf_attr.ca_pver;
 	m0_fid_convert_stob2cob(&cp2snscp(cp)->sc_stob_id, &fid);
 
-	return m0_io_cob_stob_create(&cp->c_fom, m0_sns_cm_cp2cdom(cp), &fid,
-				     pver, fctx->sf_attr.ca_lid, true, &cob);
+	ret = m0_io_cob_stob_create(&cp->c_fom, m0_sns_cm_cp2cdom(cp), &fid,
+				    pver, fctx->sf_attr.ca_lid, true, &cob);
+	if(cob != NULL)
+		m0_cob_put(cob);
```

Comment:
add a space after `if`.

If we want to check if the cob is NULL or not here, we must initialize it to NULL at the beginning.
So, please do:
```
struct m0_cob             *cob = NULL;
```
Because if the function `m0_io_cob_stob_create()` fails, the `cob` may be unmodified, and it may be a random value.

Reply:
Done.